modules/nf-core/glimpse2/chunk/main.nf

```diff
@@ -0,0 +1,50 @@
+process GLIMPSE2_CHUNK {
+    tag "$meta.id"
+    label 'process_low'
+
+    beforeScript  """
+        if cat /proc/cpuinfo | grep avx2 -q
+        then
+            echo "Feature AVX2 present on host"
+        else
+            echo "Feature AVX2 not present on host"
```

Comment:
Is the resulting error message clear that this module is the trigger?

Reply:
The error look like this
```
Caused by:
  Process `test_shapeit5_phaserare:SHAPEIT5_PHASERARE (input)` terminated with an error exit status (1)

Command executed:

  SHAPEIT5_phase_rare \
       \
      --input-plain 1000GP.chr21.noNA12878.s.bcf \
      --scaffold input_chr21.vcf.gz \
       \
       \
      --input-region chr21:16600000-16800000 \
      --scaffold-region chr21:16600000-16800000 \
      --thread 2 \
      --output input_chr21_16600000-16800000.vcf.gz
  
  cat <<-END_VERSIONS > versions.yml
      "test_shapeit5_phaserare:SHAPEIT5_PHASERARE":
          shapeit5: "$(SHAPEIT5_phase_rare | sed -nr '/Version/p' | grep -o -E '([0-9]+.){1,2}[0-9]' | head -1)"
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  Feature AVX2 not present on node
```